vars/createGithubIssue.groovy

```diff
@@ -13,27 +13,49 @@
  @param args.issueTitle <required> - GitHub issue title
  @param args.issueBody <required> - GitHub issue body
  @param args.label <optional> - GitHub issue label to be attached along with 'untriaged'. Defaults to autocut.
+ @param args.daysToReOpen <optional> - Look for a closed Github issues older than `daysToReOpen`.
  */
+
 void call(Map args = [:]) {
     label = args.label ?: 'autocut'
+    daysToReOpen = args.daysToReOpen ?: '3'
     try {
         withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
-            def issues = sh(
-                    script: "gh issue list --repo ${args.repoUrl} -S \"${args.issueTitle} in:title\" --label ${label}",
-                    returnStdout: true
-            )
-            if (issues) {
-                println('Issue already exists, adding a comment.')
-                def issuesNumber = sh(
+            def openIssue = sh(
                     script: "gh issue list --repo ${args.repoUrl} -S \"${args.issueTitle} in:title\" --label ${label} --json number --jq '.[0].number'",
                     returnStdout: true
-                ).trim()
+            ).trim()
+
+            def currentDayMinusThree = sh(
```

Comment:
Maybe rename this variable. Sounds confusing. 
```suggestion
            def currentDayMinusDaysToReOpen = sh(
```

Reply:
Looks like this is a NIT, the variable `currentDayMinusThree` is a `Date` and then used in a search query. 
The name `currentDayMinusThree` makes more sense as the output of this variable is `Date` but this `currentDayMinusDaysToReOpen` does not look like `Date`.

@gaiksaya  how about `dateThreeDaysAgo` ?

Reply:
My point is `Three` might not be true right? Since we are taking arg from the user too.

Reply:
Updated @gaiksaya